charts/controller/values.yaml

```diff
@@ -6,6 +6,13 @@ image:
 service:
   type: ClusterIP
 
+ingress:
+  enabled: False
+  host: ''
```

Comment:
Double check having an empty string in the values does not impact the `default` host the ingress should have

Reply:
With a above default set, the blank is read as a null so the default argument is used whatever that might be. Which is the expected behavior.